src/agent/src/config.rs

```diff
@@ -21,6 +21,8 @@ const DEFAULT_HOTPLUG_TIMEOUT: time::Duration = time::Duration::from_secs(3);
 const DEFAULT_CONTAINER_PIPE_SIZE: i32 = 0;
 const VSOCK_ADDR: &str = "vsock://-1";
 const VSOCK_PORT: u16 = 1024;
+
+// Environment variable used for development and testing
```

Comment:
nit: should this commit be squashed? (sorry; GitHub isn't good at providing commit reviews)